activestorage/lib/active_storage/engine.rb

```diff
@@ -73,6 +73,7 @@ class Engine < Rails::Engine # :nodoc:
         ActiveStorage.analyzers         = app.config.active_storage.analyzers || []
         ActiveStorage.paths             = app.config.active_storage.paths || {}
         ActiveStorage.routes_prefix     = app.config.active_storage.routes_prefix || "/rails/active_storage"
+        ActiveStorage.draw_routes       = app.config.active_storage.draw_routes
```

Comment:
We need to document this option in the configuring guide.

Reply:
Done!